lib/search/query_components/core_query.rb

```diff
@@ -71,7 +71,7 @@ def unquoted_phrase_query
           match_all_terms(%w(title acronym description indexable_content)),
           match_any_terms(%w(title acronym description indexable_content)),
           minimum_should_match("all_searchable_text")
-        ])
+        ], tie_breaker: 0.7)
```

Comment:
https://www.elastic.co/guide/en/elasticsearch/reference/6.7/query-dsl-dis-max-query.html 👍

Reply:
Magic numbers straight from the horse's mouth